backend/infrahub/core/schema/definitions/core.py

```diff
@@ -1317,6 +1317,7 @@
             "attributes": [
                 {"name": "conflicts", "kind": "JSON"},
                 {"name": "keep_branch", "enum": BranchConflictKeep.available_types(), "kind": "Text", "optional": True},
+                {"name": "enriched_conflict_id", "kind": "Text", "optional": True},
```

Comment:
I don't think this requires a migration b/c CoreDataChecks are not long-lived objects and the new field is only used with the new diff